src/main/webapp/availability.js

```diff
@@ -38,3 +38,17 @@ function browserTimezoneOffset() {
 function availabilityTableDiv() {
   return document.getElementById('table-container');
 }
+
+function updateAvailability() {
+  let selectedSlots = document.getElementsByClassName('table-success');
+  let firstSlot = document.getElementsByTagName('tbody').item(0)
+    .firstElementChild.firstElementChild.getAttribute('data-utc');
+  let lastSlot = document.getElementsByTagName('tbody').item(0)
+    .lastElementChild.lastElementChild.getAttribute('data-utc');
+  let utcTimestamps = [
+    firstSlot,
+    lastSlot
+  ].concat(Array.from(selectedSlots).map(s => s.getAttribute('data-utc')));
+  
+  let request = new Request('/availability', {method:'PUT', body:JSON.stringify(utcTimestamps)});
```

Comment:
I think there should be spaces after each of the colons here.

Reply:
Done